### PR TITLE
frontend readiness: Abhi

### PR DESCRIPTION
**Pull request by forebitt**
1. add schema validation on saving data on-chain
2. fix local metadata key

### CODE REVIEW

File: apps/web/src/lib/schemaValidator.ts [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+    addFormats(ajv);
+    ajv.addSchema(this.schema, this.schema_name);
+    ajv.addFormat("url", /^https?:\/\/.+/);
+    const validate = ajv.compile(this.schema);
+    const valid = validate(data);
+    return [valid, JSON.stringify(validate.errors)];
+  }
+}
+
+const validExample = {
```
Comment (rishikesh-y) [inline apps/web/src/lib/schemaValidator.ts:33]: remove this examples from the file

Comment (forebitt) [reply apps/web/src/lib/schemaValidator.ts:33]: kept the examples for the future reference.

Comment (r4881t) [reply apps/web/src/lib/schemaValidator.ts:33]: Naah it's not needed really. You can remove example and just add comments on usage if you feel like,

Comment (forebitt) [reply apps/web/src/lib/schemaValidator.ts]: removed the examples

**Review by r4881t — APPROVED** [review]

**Pull request merged**